operator/v1/types_ingress.go

```diff
@@ -633,8 +636,19 @@ type AWSClassicLoadBalancerParameters struct {
 }
 
 // AWSNetworkLoadBalancerParameters holds configuration parameters for an
-// AWS Network load balancer.
+// AWS Network load balancer. For Example: Setting AWS EIPs https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html
 type AWSNetworkLoadBalancerParameters struct {
+	// You can assign Elastic IP addresses to the Network Load Balancer.
```

Comment:
same comments apply from config/v1/types_ingress.go

Reply:
sure made the changes